### PR TITLE
monitor

### LINKED ISSUE #290: Implement the monitor plugin for the HuaweiCloud (with `cpu/mem` for VM).
- [ ] Implement the monitor plugin for the HuaweiCloud with cpu/mem for VM.

### PR DESCRIPTION
**Pull request by wanglin100**
fixed #290

### CODE REVIEW

File: plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/HuaweiCloudOrchestratorPlugin.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public List<AbstractCredentialInfo> getCredentialDefinitions() {
 
     @Override
     public List<Metric> getMetrics(AbstractCredentialInfo credential,
-            DeployResource deployResource) {
+            DeployResource deployResource, String regionName) {
```
Comment (iskey) [inline plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/HuaweiCloudOrchestratorPlugin.java:67]: must we pass through the region?

Comment (wanglin100) [reply plugins/huaweicloud/src/main/java/org/eclipse/xpanse/orchestrator/plugin/huaweicloud/HuaweiCloudOrchestratorPlugin.java]: ok

File: modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/MonitorServiceInfo.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.monitor.models;
```
Comment (iskey) [inline modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/MonitorServiceInfo.java:7]: seems we don't need this.

File: modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/ServiceInfo.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ServiceInfo {
+
+    /**
+     * Id of the deployed service.
+     */
+    private String id;
```
Comment (iskey) [inline modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/ServiceInfo.java:24]: ditto

File: modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/MonitorServiceInfo.java (hunk 2 above)
Comment (wanglin100) [reply modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/MonitorServiceInfo.java]: It has been modified

File: modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/ServiceInfo.java (hunk 3 above)
Comment (wanglin100) [reply modules/monitor/src/main/java/org/eclipse/xpanse/modules/monitor/models/ServiceInfo.java]: It has been modified

**Review by iskey — APPROVED** [review]
looks good.

**Pull request merged**